common/archive-impl/impl2/src/java/org/sakaiproject/archive/impl/SiteArchiver.java

```diff
@@ -148,29 +160,37 @@ public String archive(String siteId, String m_storagePath, String fromSystem)
 			root.setAttribute("date", now.toString());
 			root.setAttribute("system", fromSystem);
 			root.setAttribute("xmlns:sakai", ArchiveService.SAKAI_ARCHIVE_NS);
+			root.setAttribute("xmlns:CHEF", ArchiveService.SAKAI_ARCHIVE_NS.concat("CHEF"));
 			
 			stack.push(root);
 
+			final String serviceName = service.getClass().getCanonicalName();
 			try {
-				final String serviceName = service.getClass().getCanonicalName();
 				transactionTemplate.executeWithoutResult(
 						transactionStatus -> results
 								.append("<===== Start ")
-								.append(serviceName)
+								.append(service.getLabel())
+								.append(" [").append(serviceName).append("]")
 								.append(" =====>\n")
 								.append(service.archive(siteId, doc, stack, storagePath, attachments))
 								.append("<===== End ")
 								.append(serviceName)
-								.append(" =====>\n"));
+								.append(" =====>\n\n"));
 			}
 			catch (Throwable t)
 			{
-				results.append(t.toString() + "\n");
+				String failure = String.format("Failure archiving site %s from service %s [%s]: %s", siteId, service.getLabel(), serviceName, t.getMessage());
+				log.error(failure, t);
```

Comment:
I would advocate for these to be warning's instead of error. Warning fit's better as there were errors in processing the user's request but the request itself is not in error.
```suggestion
				log.warn(failure, t);
```